DEVELOPER_GUIDE.md

```diff
@@ -61,6 +62,19 @@ go test -v -run TestName;
 
 In order to test opensearch-go client, you need a running OpenSearch cluster. You can use Docker to accomplish this. The [Docker Compose file](.ci/opensearch/docker-compose.yml) supports the ability to run integration tests for the project in local environments. If you have not installed docker-compose, you can install it from this [link](https://docs.docker.com/compose/install/).
 
+### Composing an OpenSearch Docker Container
+
+Ensure that Docker is installed on your local machine. You can check by running `docker --version`. Next, navigate to your local OpenSearch repository where the docker-compose.yaml file is stored. Run the following command to build the docker container. 
+
+```
+cd opensearch/.ci/opensearch
+docker-compose -f docker-compose.yml up
```

Comment:
This is also handled in the Makefile. So you don't need to change folders.

```
make cluster.build
make cluster.start
```
You can also run:
```
make cluster.stop
make cluster.clean
```